rush.json

```diff
@@ -2226,6 +2226,11 @@
       "packageName": "@azure/arm-oracledatabase",
       "projectFolder": "sdk/oracledatabase/arm-oracledatabase",
       "versionPolicyName": "management"
+    },
+    {
+      "packageName": "@azure-rest/health-deidentification",
```

Comment:
In order to not block this, we can add ci and live test pipelines in a follow up PR

Reply:
Sounds good